functions.php

```diff
@@ -6,7 +6,7 @@ function scripts_and_styles_method() {
   // library.js is to bundle plugins. my.js is your scripts. enqueue more files as needed
   $jslib = $templateuri."library.js";
   wp_enqueue_script( 'jslib', $jslib,'','',true);
-  $myscripts = $templateuri."main.js";
+  $myscripts = $templateuri."main.min.js";
```

Comment:
with sourcemaps there is no real need to ever serve a non minified file so better to have it like this all the way thru dev imo